### DISCUSSION TITLE
Minifying CSS and Javascript?

### DISCUSSION THREAD

**Discussion by antforager** [bootscore/bootscore#910] [Q&A]
Pagespeed insights says my site (antgear.com) has a bunch of unminified JS and CSS files. Some of them are located inside of plugins and also  a couple in the bootscore theme. Is there a good way to minify those files?

**Comment by crftwrk** [comment]
Bootscore's and it's plugin assets are already minified. But, of course, everything what you have installed on your site adds something to your frontend. WooCommerce for example enqueues 4 CSS and 7 JS files which increases file request. You need a plugin that fetches all `.css` and `.js` files and combine them into one minified `.js` and `.css` file.

There are several plugins for that, bootscore.me uses https://wordpress.org/plugins/autoptimize/. The settings might have an impact to your site, and they must be fine-tuned to your individual needs. You can check this (outdated) post what settings are used on bootscore.me https://bootscore.me/blog/new-plugin-new-feature-speed/.

Solved?
